windows/ReactTestApp/Manifest.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <map>
+#include <optional>
+#include <string>
+#include <vector>
+
+namespace winrt::ReactTestApp::implementation
+{
+    struct Component {
+        std::string appKey;
+        std::optional<std::string> displayName;
+        std::optional<std::map<std::string, std::string>> initialProperties;
```

Comment:
It looks like this only supports string values. Initial properties supports other types as well though, as reported in #144. I don't think we should do anything now in this PR but we will have to revisit this when integrating React Native. It looks like we'll need to convert this to [`folly::dynamic`](https://github.com/microsoft/react-native-windows/blob/532e299688d745c0d7c5d8790a681f60813eb678/vnext/Microsoft.ReactNative/ReactHost/React.h#L89).